src/Workspaces/CSharp/Portable/Editing/CSharpImportAdder.cs

```diff
@@ -57,5 +67,102 @@ private INamespaceSymbol GetExplicitNamespaceSymbol(ExpressionSyntax fullName, E
 
             return null;
         }
+
+        private class Rewriter : CSharpSyntaxRewriter
+        {
+            private Workspace _workspace;
+            private CancellationToken _cancellationToken;
+            private readonly SemanticModel _model;
+            private readonly HashSet<string> _namespaceMembers;
+            private readonly HashSet<string> _extensionMethods;
+
+            public Rewriter(
+                IEnumerable<INamespaceOrTypeSymbol> namespaceMembers,
+                IEnumerable<IMethodSymbol> extensionMethods,
+                SemanticModel model,
+                Workspace workspace,
+                CancellationToken cancellationToken)
+            {
+                _model = model;
+                _workspace = workspace;
+                _cancellationToken = cancellationToken;
+                _namespaceMembers = new HashSet<string>(namespaceMembers.Select(x => x.Name));
+                _extensionMethods = new HashSet<string>(extensionMethods.Select(x => x.Name));
+            }
+
+            public override bool VisitIntoStructuredTrivia => true;
+
+            public override SyntaxNode VisitIdentifierName(IdentifierNameSyntax node)
+            {
+                //Only visit leading trivia, as we only care about xml doc comments
```

Comment:
space after //

Reply:
applies to other methods in this class as well.

Reply:
i'm confused how an identifier can have xml doc comments.

Reply:
If it's a return type.

Reply:
gotcha.  don't particularly like this (as it seems unnecessary for 99.9% of identifiers).  Can we instead do this for specific nodes that can have doc comments?

Reply:
note: this isn't hugely important.  but i thought maybe a few brain cycles on this might be nice.